paymentsheet/src/test/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEventTest.kt

```diff
@@ -421,11 +425,13 @@ class PaymentSheetEventTest {
             "allows_delayed_payment_methods" to true,
             "appearance" to expectedAppearance,
             "billing_details_collection_configuration" to expectedBillingDetailsCollection,
+            "is_server_side_confirmation" to false,
```

Comment:
I didn't see any where it was true, do we want to test that as well?

Reply:
The main logic is happening in the `FlowControllerConfigurationHandler` and `PaymentSheetViewModel`, which is why those are most extensively tested. I don’t see a lot of benefit testing things here.

Reply:
Sounds good to me.